obspy/core/inventory/response.py

```diff
@@ -166,9 +166,6 @@ def __str__(self):
                 if self.decimation_input_sample_rate is not None else ""))
         return ret.strip()
 
-    def _repr_pretty_(self, p, cycle):
```

Comment:
Did this get moved down a layer or something? I'm not sure how this relates to `AttribDict`.